GetIntoTeachingApi/Models/TeachingEvent.cs

```diff
@@ -68,6 +68,7 @@ public enum EventType
         [EntityRelationship("msevtmgt_event_building", typeof(TeachingEventBuilding))]
         public TeachingEventBuilding Building { get; set; }
         [JsonIgnore]
+        [EntityField("msevtmgt_building", typeof(EntityReference), "msevtmgt_buildingid")]

```

Comment:
Can we add a test for this attribute? there should be others you can see in `TeachingEventTests` that are tested to copy